webpage/src/components/CategoryTile.js

```diff
@@ -33,12 +33,17 @@ const CategoryTitle = ({
         {comingSoon ? (
           <NotAvaliable text="Coming Soon" showContact />
         ) : (
-          <Img
-            className="my-auto"
-            alt="product"
-            fixed={images?.[0]?.fixed ? images[0].fixed : null}
-            fluid={images?.[0]?.fluid ? images[0].fluid : null}
-          />
+          <div className="relative w-full h-0 aspect-ratio-square">
```

Comment:
I must admit this css confuses me.

"h-0" makes the height 0, how does the inner div with the image manage to override this and display?

I need to set up a tailwindcss playground project just to play and learn this stuff.

Reply:
It's definitely not intuitive, but here's how it works:
We want to make sure that the element has no implicit height, the only vertical space it should take up is the padding.
The `aspect-ratio-square` class just adds: `padding-bottom: 100%`.
I could have just extended the padding scale to include `100%`, but I think it's more readable this way, and we can also extend the scale to include other sized rectangles (useful for iframes).

Reply:
far out @lukebennett88 your my CSS hero!!!!
If you make a CSS course, or product I'll be your first customer!! take my money!!